BookingsApi/BookingsApi.IntegrationTests/Features/CaseTypes.feature

```diff
@@ -13,7 +13,7 @@ Feature: CaseTypes
       | CaseTypeName                      | HearingTypeName                                                                                                                                                                                                                                                                                                                                            |
       | Civil Money Claims                | Application to Set Judgment Aside,First Application,Directions Hearing,Case Management Hearing,Final Hearing                                                                                                                                                                                                                                               |
       | Financial Remedy                  | First Directions Appointment,First Application,Directions Hearing,Case Management Hearing,Hearing,Final Hearing,Financial Dispute Resolution,Interim hearing                                                                                                                                                                                                                                            |
-      | Generic                           | Hearing                                                                                                                                                                                                                                                                                                                                                    |
+      | Generic                           | Hearing,Daily Test,Demo,Familiarisation,One to one,Test                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
please can you add 'Automated Test'? Emma has updated the story

Reply:
Done